run_sets/testsuite.yml

```diff
@@ -18,16 +19,36 @@
 - features/centos_tradclient.feature
 - features/centos_salt.feature
 - features/centos_salt_ssh.feature
+
 ## CORE feature end ###
 
 # SECONDARY FEATURE:
 
-# this feature can be runned in random order.
-# FIXME: MAKE THIS HAPPEN
+# PREFIX:
+
+# type (mandatory)
+# - srv (feature testing server side)
+# - min (feature testing minions sles)
+# - centos (feature testing centos/rhes)
+# - trad_cli (feature testing tradional client)
+# - allcli (feture testing trad + minion(centos/sles)
+
+# youn can combine the prefix.
+
+# topic (for salt mandatory) (sometimes is optional)
+# - salt (feature concering salt)
+# - docker
+
+# EXAMPLES
+
+# * srv_salt_myname.feature
+# * srv_min_salt_bootstrap.feature
```

Comment:
I wouldn't allow to combine the prefix, because if so, you would also need to set an prefix order, to avoid somethign like this:
```
srv_min_salt_bootstrap.feature
salt_min_srv_whatever.feature
min_salt_another_whatever.feature
docker_min_srv_whatever.feature
```

So at the end we would have a mess again 😄 
What do you think?